scripts/import_open_textbook_library.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+import json
+import requests
+import time
+from typing import Any
+from openlibrary.core.imports import Batch
+from scripts.solr_builder.solr_builder.fn_to_cli import FnToCLI
+from openlibrary.config import load_config
+
+FEED_URL = 'https://open.umn.edu/opentextbooks/textbooks.json?'
+
+
+def get_feed():
+    """Fetches and returns the feed."""
+    r = requests.get(FEED_URL)
+    return r.json()
+
+
+def map_data(data) -> dict[str, Any]:
+    import_record = {}
```

Comment:
This might fix the type issues.
```suggestion
    import_record: dict[str, Any] = {}
```